print_env.sh

```diff
@@ -29,19 +30,19 @@ lscpu
 echo
 
 echo "***CMake***"
-which cmake && cmake --version 
+which cmake && echo cmake --version 
```

Comment:
What's with the `echo`? That will just cause it to print the command, rather than the output of the command, won't it?

Reply:
@harrism not when I ran it locally ([result here](https://gist.github.com/trxcllnt/4843c15a483de805e7b39db740014271)). The issue I saw was `which cmake` prints a newline after the path, but `read` only sees one a single statement, and doesn't indent the output of `cmake --version`. The `echo` on the second statement gets read as a new line in the indent loop at the bottom.

Reply:
```
mharris@dgx14:~/gitlab-nvidia/RAPIDS/cudf/cpp/build$ which cmake && echo cmake --version
/usr/local/bin/cmake
cmake --version
```

Reply:
Haha whoops I was looking at the wrong window. Fixed now!